menyer/enmeny/style1.css

```diff
@@ -0,0 +1,52 @@
+body {
+    background-image: url('../enmeny/images/wallpaper.jpg');
+    font-family: 'Times New Roman', Times, serif;
+    color: aqua;
+}
+
+header {
+    font-style: inherit;
+    font-family: monospace;
+    font-size: 20px;
+}
+
+.forratter {
+    background-color: aqua;
+    padding: 20px;
+    margin: 200px;
+    border-radius: 30px;
+    color: blueviolet;
+    box-shadow: 13px 13px 8px 8px magenta;
+}
+
+.skinkis, .fruktis, .julis, .paris, .kottis, .cremis {
+    border-style: dotted;
+    border-width: 3px;
+    border-radius: 30px;
+}
+
+.huvudratter {
+    background-color: aqua;
+    padding: 20px;
+    margin: 200px;
+    border-radius: 30px;
+    color: blueviolet;
+    box-shadow: 13px 13px 8px 8px magenta;
+}
+
+.efterratter {
+    background-color: aqua;
+    padding: 20px;
+    margin: 200px;
+    border-radius: 30px;
+    color: blueviolet;
+    box-shadow: 13px 13px 8px 8px magenta;
+}
```

Comment:
Looks like the same styling. I would just have one class called `.menu-section`.